docs/guides/multisig-wallets/create-multisig.md

```diff
@@ -62,6 +62,13 @@ Log in to the Camino Wallet using one of the member accounts you intend to inclu
 
 - **Multisignature Name**: This names your multisig wallet and gets recorded on the blockchain. Ensure it doesn't contain any private or sensitive information.
 - **Multisignature Co-owners**: Lists members of the multisig wallet. The initial member is the account you've logged in with. Add more members by clicking the green '+' icon. Note: names in this section are stored in your browser's local cache and aren't blockchain entries.
+
+:::caution AVOID USING MULTISIG ALIASES FOR CO-OWNER ADDRESSES
+
+It's important to understand that other multisig aliases should not be entered as member addresses here. Nested multisigs are not supported and can lead to unintended issues.
```

Comment:
Instead of **should** I'd use the word **must** not be entered.

And the *unintended issues* are pretty simple to explain:
The implementation of MultiSig Aliases is, that it can only be signed by single sig accounts. Therefore signing a pending request of a MultiSig Alias with a MultiSig Alias as a member is simply not possible in the Frontend.

Reply:
I incorporated the explanation into the warning message. What do you think?